packages/framework-integration-tests/integration/end-to-end/cart.integration.ts

```diff
@@ -325,6 +325,175 @@ describe('Cart end-to-end tests', () => {
         expect(productData).to.be.deep.equal(expectedResult)
       })
     })
+
+    context('Deletions', () => {
+      let mockSku: string
+      let mockDisplayName: string
+      let mockDescription: string
+      let mockPriceInCents: number
+      let mockCurrency: string
```

Comment:
Why not:
```suggestion
      let mockProduct : {
        sku: string,
        displayName: string,
        description: string,
        priceInCents: number,
        currency: string
      }
``` 

So then you can do... (see next comments)

Reply:
I agree, let's change that.

Reply:
Fixed [here](https://github.com/boostercloud/booster/pull/407/commits/a165f7a37c4b48185a08ff610c08f48e7941459e)